### DISCUSSION TITLE
Help! readonly on Pro slider with linked inputs

### DISCUSSION THREAD

**Discussion by idv8** [formkit/formkit#957] [General]
I intend to use the[ Pro slider component with linked inputs](https://formkit.com/inputs/slider#linked-input-attributes) but need to bind "readonly" to the inputs so that they cannot be edited. 

So the inputs would display the changing values from moving the handles, which happens out of the box, but could not be directly clicked and edited via keyboard as is their default behavior when the show-input prop is used. 

Your help would be appreciated! Thanks in advance!

**Comment by fenilli** [comment]
If you don't intend for those inputs do be changeable why do you need it? wouldn't tooltip visibility a better candidate for that?

Otherwise you can pass `disabled` and `readonly` to `min-input-attrs` and `max-input-attrs` too.

**Reply by idv8** [reply]
You're right except I'm working a mockup that's been approved at many levels above me. We'll actually be hiding the tooltip, but you're so right. Thanks for the help!
